components/form-category-input/index.js

```diff
@@ -25,11 +27,13 @@ export default function CategoryInput({ FxCategories }) {
               value={category}
               checked={selectedCategories.includes(category)}
               onChange={handleCategoryChange}
+              //onCancel={onCancel}
```

Comment:
Clean Code rules - please remove this comment before the merge 🧹

```suggestion
         
```

Reply:
done